packages/reactotron-server/src/server/apollo/index.ts

```diff
@@ -1,13 +1,21 @@
 import { ApolloServer } from "apollo-server-express"
+import { buildSchema } from "type-graphql"
 
-import typeDefs from "./schema"
-import resolvers from "./resolvers"
+import { messaging } from "../messaging"
+import { resolvers } from "./resolvers"
 
 let apolloServer: ApolloServer
 
-export function createApolloServer() {
+export async function createApolloServer() {
```

Comment:
Perhaps it would be good to establish a convention for returning singletons? Something like `getApolloServer` or just `apolloServerInstance`?

Reply:
For context, I like the way Objective-C does this -> `SomeClass *instance = [SomeClass sharedInstance]`

Reply:
I was thinking about that as I was wrapping up last night. I was also wondering if we need some sort of wrappers around ApolloServer or the Reactotron server to attach methods that are useful from the server side. I will be playing around with ideas here.